src/Response/InitiatePaymentResponse.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright (c) 2016 Martin Aarhof
```

Comment:
Is this really the case?

Reply:
Updated the copyright with AltaPay and the current year.